src/common/fs/FileUtils.cpp

```diff
@@ -95,7 +93,6 @@ StatusOr<std::string> FileUtils::readLink(const char *path) {
     return std::string(buffer, len);
 }
 
-
```

Comment:
@sherman-the-tank encourages this styling. Yet this has not been synced to others.